config/controller.js

```diff
@@ -33,18 +33,18 @@ module.exports = {
   isLoggedIn: (req, res, next) => {
```

Comment:
Please remove this function and make sure the API its still working, we don't need this function any more